pom.xml

```diff
@@ -37,9 +37,9 @@
         <digital-twin-device-client-artifact-id>digital-twin-device-client-preview</digital-twin-device-client-artifact-id>
         <digital-twin-service-client-artifact-id>digital-twin-service-client-preview</digital-twin-service-client-artifact-id>
 
-        <iot-device-client-version>1.0.1</iot-device-client-version>
-        <iot-service-client-version>1.0.1</iot-service-client-version>
-        <iot-deps-version>1.0.1</iot-deps-version>
+        <iot-device-client-version>1.1.0</iot-device-client-version>
+        <iot-service-client-version>1.1.0</iot-service-client-version>
+        <iot-deps-version>1.1.0</iot-deps-version>
         <provisioning-device-client-version>1.0.1</provisioning-device-client-version>
```

Comment:
Why is deps major bump?

Reply:
It added new APIs for the file upload granular APIs

Reply:
https://github.com/Azure/azure-iot-sdk-java/commit/54769d2373fda4281207d77c6ec33d3623e46252

Reply:
👍

Reply:
make sure you mention this addition in release notes.